### ISSUE TITLE
Will not connect to server.

### ISSUE THREAD

**Issue by TechCallCP** [MCXboxBroadcast/Broadcaster#163]
The plugin and standalone versions are refusing to connect players who join through it to said geysermc server, however players who directly connect through GeyserMC get instantly connected no issues. It also seems to time out while connecting. I enabled debug mode to give console info on issue and what it looks like is Icej4 is making a constant loop triggering checks, but not pushing past it. I attempt to connect 3 times in the log, and the second time was the only one successful. It also took around 30-45 seconds for it to start connecting on second attempt. Log is included below.
[2024-11-10-7.log.gz](https://github.com/user-attachments/files/17694576/2024-11-10-7.log.gz)

**Comment by TechCallCP** [comment]
> Have you altered the configuration to put in the already connectable IP address?

Yes, I have tried the direct IP address and the domain its connected through. Both do the exact same thing. I noticed another person had a similar issue where they couldn't connect through this but could directly through geyser. Both the plugin version and standalone versions do the exact same thing.

**Comment by rtm516** [comment]
Could be your host blocking the way the tool works, I suggest joining discord so we can get this resolved faster

**Comment by rtm516** [comment]
No update from OP or other users with issue, closing.
